docs/changelogs/CHANGELOG-2.25.md

```diff
@@ -4,8 +4,27 @@
 - [v2.25.1](#v2251)
 - [v2.25.2](#v2252)
 - [v2.25.3](#v2253)
+- [v2.25.4](#v2254)
 
-## [v2.25.3](https://github.com/kubermatic/kubermatic/releases/tag/v2.25.3)
+
+## v2.25.4
+
+**GitHub release: [v2.25.4](https://github.com/kubermatic/kubermatic/releases/tag/v2.25.4)**
+
+### Bugfixes
+
+- Fix #13393 where externally deployed Velero CRDs are removed automatically from user user cluster ([#13396](https://github.com/kubermatic/kubermatic/pull/13396))
```

Comment:
Please turn this into a link, outside of GitHub no markdown renderer will understand how to make # 13393 usable.